docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md

```diff
@@ -0,0 +1,361 @@
+kube-system   pod/canal-w4bwm                                   2/2     Running   0          57s
+kube-system   pod/coredns-d6d746d5d-9d4kn                       1/1     Running   0          2m40s
+kube-system   pod/coredns-d6d746d5d-wrcbz                       1/1     Running   0          2m40s
+kube-system   pod/etcd-u1a1.ubuntu.lh                           1/1     Running   0          2m49s
+kube-system   pod/etcd-u1a2.ubuntu.lh                           1/1     Running   0          2m7s
+kube-system   pod/etcd-u1a3.ubuntu.lh                           0/1     Pending   0          1s
+kube-system   pod/kube-apiserver-u1a1.ubuntu.lh                 1/1     Running   0          2m49s
+kube-system   pod/kube-apiserver-u1a2.ubuntu.lh                 1/1     Running   0          2m7s
+kube-system   pod/kube-apiserver-u1a3.ubuntu.lh                 1/1     Running   0          24s
+kube-system   pod/kube-controller-manager-u1a1.ubuntu.lh        1/1     Running   1          2m49s
+kube-system   pod/kube-controller-manager-u1a2.ubuntu.lh        1/1     Running   0          2m7s
+kube-system   pod/kube-proxy-dk9k8                              1/1     Running   0          2m19s
+kube-system   pod/kube-proxy-dnkx5                              1/1     Running   0          64s
+kube-system   pod/kube-proxy-q4h7x                              1/1     Running   0          57s
+kube-system   pod/kube-proxy-s2wzx                              1/1     Running   0          84s
+kube-system   pod/kube-proxy-vddsl                              1/1     Running   0          64s
+kube-system   pod/kube-proxy-vnhlm                              1/1     Running   0          2m40s
+kube-system   pod/kube-scheduler-u1a1.ubuntu.lh                 1/1     Running   1          2m48s
+kube-system   pod/kube-scheduler-u1a2.ubuntu.lh                 1/1     Running   0          2m7s
+kube-system   pod/kube-scheduler-u1a3.ubuntu.lh                 1/1     Running   0          8s
+kube-system   pod/machine-controller-7bfb85845-n4f5g            1/1     Running   0          56s
+kube-system   pod/machine-controller-webhook-7bf5f89954-47qp7   1/1     Running   0          56s
+kube-system   pod/metrics-server-5f75c7cb4f-gj4rq               1/1     Running   0          81s
+kube-system   pod/node-local-dns-54vxf                          1/1     Running   0          64s
+kube-system   pod/node-local-dns-7h5m8                          1/1     Running   0          64s
+kube-system   pod/node-local-dns-7tsvf                          1/1     Running   0          82s
+kube-system   pod/node-local-dns-c58zf                          1/1     Running   0          82s
+kube-system   pod/node-local-dns-dvnbb                          1/1     Running   0          57s
+kube-system   pod/node-local-dns-vsjpb                          1/1     Running   0          82s
+```
+
+### 5.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated (but provides at least a script to download docker images)
```

Comment:
can you please provide links to resources explaining those two things?

Reply:
added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation